front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -43,6 +46,30 @@ const ManageTrainSchedule = ({ trainIdToEdit }: ManageTrainScheduleProps) => {
   const origin = useSelector(getOrigin);
   const destination = useSelector(getDestination);
   const pathSteps = useSelector(getPathSteps);
+
+  const pathStepWithPointType = useMemo(
+    () =>
+      pathSteps.map((step, index, array) => {
+        if (!step) return null;
+
+        let pointType;
```

Comment:
nit: if you want to save 1 line, you can initiate `pointType` here and remove the last test
```suggestion
        let pointType = MARKER_TYPE.VIA;
```

Reply:
Done